### PR TITLE
Create skeleton

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by rudigiesler** [comment]
Ready for review

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    license='BSD',
+    description="An HTTP API for interacting with vumi",
+    long_description=open('README.md', 'r').read(),
+    author='Praekelt Foundation',
+    author_email='dev@praekeltfoundation.org',
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
+        'vumi',
+        'txJSON-RPC',
```
Comment (hodgestar) [inline setup.py]: Do we need `txJSON-RPC`? I'm hoping not.

**Comment by hodgestar** [comment]
One small question, otherwise looks good.

File: setup.py (hunk 1 above)
Comment (rudigiesler) [inline setup.py]: No we don't. I was looking at the wrong thing initially, and forgot to remove this. Will remove.

**Comment by rudigiesler** [comment]
Changes made. Ready for re-review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**